### ISSUE TITLE
Table header is rendered incorrectly on mobile

### ISSUE THREAD

**Issue by rclmenezes** [fomantic/Fomantic-UI#787]
# Bug Report

Table header is rendered incorrectly on mobile. The border doesn't go all the way and it gets a weird double border on the left.

## Steps to reproduce
Go to a table with a narrow screen or on a mobile device. The docs, for example: https://fomantic-ui.com/collections/table.html

## Expected result

This is how it renders on [Semantic UI](https://fomantic-ui.com/collections/table.html):

![image](https://user-images.githubusercontent.com/736775/58654992-1c939100-82e7-11e9-8290-229c54315cea.png)

## Actual result

This is how it renders on [Fomantic](https://fomantic-ui.com/collections/table.html):

![image](https://user-images.githubusercontent.com/736775/58654899-f0781000-82e6-11e9-8fb3-a87b91942805.png)

**Comment by rclmenezes** [comment]
Looks like the following CSS fixes this:

```
table.ui.celled thead tr {
  width: auto !important;
  display: block !important;
}

table.ui.celled thead tr th {
  border-left: none !important;
}
```

**Comment by 5t4r8ucK** [comment]
Didn't fix it for me. I used this instead:
`@media only screen and (max-width: 767px) {
	.ui.table:not(.unstackable) thead tr {
		width: auto;
		display: block;
	}
	.ui.table thead tr th {
		border-left: none !important;
	}
}`

**Comment by lubber-de** [comment]
Thanks for reporting.
It is fixed by #794 
See https://jsfiddle.net/4gozw610/
